dev/examples-to-tests.el

```diff
@@ -3,9 +3,12 @@
 (require 'dash-functional)
 
 (defun example-to-should (example)
-  (let ((actual (car example))
-        (expected (nth 2 example)))
-    `(should (equal ,actual ,expected))))
+  (-let [(actual sym expected) example]
```

Comment:
Is it a good idea to have a test suite that is dependent on the code it is testing? :)